webui/react/src/shared/configs/eslintrc.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
         typeLiterals: { order: 'alphabetically' },
       },
     ],
+    '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
i think you can use `error` cuz we can use annotation to disable it anyways

Reply:
Well.. since this config is shared, maybe I should go through someone on the SAAS team to use error?

Reply:
ahh thats true.